src/excmds.ts

```diff
@@ -928,28 +933,27 @@ export function addJump(scrollEvent: UIEvent) {
     }
     const pageX = scrollEvent.pageX
     const pageY = scrollEvent.pageY
-    // Get config for current page
-    curJumps().then(alljumps => {
+    // Prevent pending jump from being registered
+    clearTimeout(JUMP_TIMEOUTID)
+    // Schedule the registering of the current jump
+    JUMP_TIMEOUTID = setTimeout(async () => {
```

Comment:
I get

```
ERROR in [at-loader] ./src/excmds.ts:939:5
    TS2322: Type 'number' is not assignable to type 'Timeout'.
```

when trying to build. Are you sure this works?

Reply:
Something must be wrong with my setup because my vscode picked up nodejs types and I thought that was normal. I will change it to `number`

Reply:
Adding this to tsconfig compilerOptions helps in my case: `    "types": ["web-ext-types"]` but I don't know if it's something you want me to add in this one? I checked that nodejs scripts like `gen_metadata.ts` still typecheck correctly with this option

Reply:
Sure, add it.